py3status/modules/speedtest_cli.py

```diff
@@ -0,0 +1,302 @@
+# -*- coding: utf-8 -*-
+"""
+Do a bandwidth test with speedtest-cli.
+
+Configuration parameters:
+    button_refresh: button to run the check (default 1)
```

Comment:
All `button_refresh` should be `2`.... Consistency across all modules.

Reply:
Ok